src/libraries/Common/tests/StreamConformanceTests/System/IO/StreamConformanceTests.cs

```diff
@@ -2938,6 +2938,72 @@ public override ValueTask<int> ReadAsync(Memory<byte> buffer, CancellationToken
         }
     }
 
+    /// <summary>Base class for a connected stream that can have writes completed.</summary>
+    public abstract class DuplexConnectedStreamConformanceTests : ConnectedStreamConformanceTests
```

Comment:
What about a test for calling Write after CompleteWrites?

Reply:
Also, a test that CompleteWrites implicitly does a Flush? (i.e. call Write without Flush, then CompleteWrites and verify the unflushed data was sent)